src/main/java/com/microsoftopentechnologies/windowsazurestorage/WAStoragePublisher.java

```diff
@@ -158,13 +156,13 @@ public void setBlobProperties(AzureBlobProperties blobProperties) {
     }
 
     @DataBoundSetter
-    public void setCntPubAccess(boolean cntPubAccess) {
-        this.cntPubAccess = cntPubAccess;
+    public void setPubAccessible(boolean pubAccessible) {
+        this.pubAccessible = pubAccessible;
```

Comment:
It does have some effects but it's no that bad. For existing jobs that already built successfully,  the container was already created using old settings and won't be created again even if this field renamed. While user update the job configuration with our new version of plugin, the new field will take effect.  The only bad effect I can image is that someone create job with old version but never build successfully before upgrade to newer version. But that's not critical as well and can be fixed easily by user.